tests/unit/test_graphstates.py

```diff
@@ -82,8 +83,13 @@ def test_index(self, random_graph):
     # def test_slice_coords(self):
     # pass
 
-    # def test_draw(self):
-    # pass
+    def test_draw(self):
```

Comment:
In reality this is testing the draw_EGraph in the viz module, so we should move this tests to the viz test files.